src/pages/commons/common-components.jsx

```diff
@@ -172,7 +174,7 @@ export const CustomAppLayout = forwardRef((props, ref) => {
       }}
     />
   );
-})
+});
```

Comment:
It looks like some changes were not formatted properly. I ran `npm run lint` locally and got this